### PR TITLE
Update defense_evasion_execution_windefend_unusual_path.toml

### PR DESCRIPTION
**Pull request by LaZyDK**
## Summary
Add Microsoft Security Client to exclusions.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_execution_windefend_unusual_path.toml [hunk 1]
```diff
@@ -32,7 +32,9 @@ process where event.type == "start" and
   (process.name : "MsMpEng.exe" and not
         process.executable : ("?:\\ProgramData\\Microsoft\\Windows Defender\\*.exe",
                               "?:\\Program Files\\Windows Defender\\*.exe",
-                              "?:\\Program Files (x86)\\Windows Defender\\*.exe"))
+                              "?:\\Program Files (x86)\\Windows Defender\\*.exe",
+                              "?:\\Program Files\\Microsoft Security Client\\*.exe",
+                              "?:\\Program Files (x86)\\Microsoft Security Client\\*.exe"))
```
Comment (austinsonger) [inline rules/windows/defense_evasion_execution_windefend_unusual_path.toml:37]: Support for Microsoft Security Client ended awhile ago.

Comment (LaZyDK) [reply rules/windows/defense_evasion_execution_windefend_unusual_path.toml:37]: I see that elastic agent is supported on both Windows 7 and Windows 8.1.

Comment (austinsonger) [reply rules/windows/defense_evasion_execution_windefend_unusual_path.toml:37]: The only thing is that Microsoft stopped support for Windows 7 or Windows 8.1 OS as well. But I will tagged @brokensound77 on this one.

Comment (brokensound77) [reply rules/windows/defense_evasion_execution_windefend_unusual_path.toml:37]: Tagging @Samirbous as the original rule author, to chime in with his thoughts

**Review by Samirbous — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
Thanks for the contribution. If you could bump the `updated_date`, we should be good to go

**Comment by LaZyDK** [comment]
`updated_date` has been updated. Thanks.

File: rules/windows/defense_evasion_execution_windefend_unusual_path.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 creation_date = "2021/07/07"
 maturity = "production"
-updated_date = "2021/07/07"
+updated_date = "2021/09/22"
 
 [rule]
 author = ["Elastic"]
```
Comment (austinsonger) [inline rules/windows/defense_evasion_execution_windefend_unusual_path.toml:7]: I think this is a big enough addition to the rule

```
author = ["Elastic", "Dennis Perto"]
```

Comment (austinsonger) [inline rules/windows/defense_evasion_execution_windefend_unusual_path.toml:7]: I think this is a big enough addition to the rule

```
author = ["Elastic", "Dennis Perto"]
```

Comment (brokensound77) [reply rules/windows/defense_evasion_execution_windefend_unusual_path.toml:7]: Makes sense to me @LaZyDK

Comment (w0rk3r) [reply rules/windows/defense_evasion_execution_windefend_unusual_path.toml:7]: @LaZyDK, can you update this one so we can merge? Thanks!

**Pull request merged**